### ISSUE TITLE
Error when retraining the object model

### ISSUE THREAD

**Issue by UV1999** [tensorflow/models#9355]
This execution is for retraining an object detection model to recognize a set of classes using our very own custom dataset. All the prerequisites were downloaded and requirements were met. And as given in the tutorial, after starting the docker container, the download and configure the training data step given in the official Google-coral tutorial were skipped because we are using our very own custom dataset.

And by following the instructions given in the Google-coral website for configuring your own training data, we took our own images and labeled all those using LableMe to get the annotations in xml format. Then we converted the same to tf_record which is required to start training the model. 

There are a total of three classes namely person, mask, and face for which we also updated the label_map.pbtxt file. The model that we used here is the MobileNet SSD V1. The model was downloaded and we also changed the pipeline.config file pointing to our own custom tf_record, label_map.pbtxt file, and the mobilenet_ssd_v1 model checkpoint file.

1. When executing the ./retrain_detection_model.sh --num_training_steps ${NUM_TRAINING_STEPS} --num_eval_steps ${NUM_EVAL_STEPS},  after the initial stages, this warning arises stating that the Variable [global_steo] is not available in checkpoint.

![1](https://user-images.githubusercontent.com/29156063/95641721-861dd500-0ac1-11eb-90b4-53a43171e412.jpg)

2. And just after executing the warning, the next error that arises is that Your CPU supports instructions that this TensorFlow binary was not compiled to use.

![2](https://user-images.githubusercontent.com/29156063/95641726-8cac4c80-0ac1-11eb-9b45-a6d6545cdb67.jpg)

3. Finally, during the initial stages of training, there is this warning repeated twice after which the execution stops. The error reads that the Allocation of 737280000 exceeds 10% of system memory.

![3](https://user-images.githubusercontent.com/29156063/95641738-9d5cc280-0ac1-11eb-8d1d-51919799af04.jpg)

**Comment by ravikyram** [comment]
@UV1999 

Can you please share colab link or simple standalone code to reproduce the issue in our environment.It helps us in localizing the issue faster.Thanks!

**Comment by UV1999** [comment]
I executed the same commands as given in just the official Docker documentation. The only difference is that I annotated 3500 images in CVAT and downloaded the TFRecord from CVAT. Then configured the pipeline.config file of MobileNet SSD V1 to point to my custom downloaded TFRecord and label_map.pbtxt file. If you need, I could share the drive link for the dataset, pipeline.config file, and label_map.pbtxt file.

And I would also like to know if this error has got something to do with my PC specs and dataset size that I have taken for it. So, can you let me know that if my configuration is not suited for this training and how could I change the dataset to run training on my PC?

PC Specs:
Intel i7 7500U - 2 cores & 4 threads
8 GB DDR4 RAM
4 GB Nvidia GTX 950M Dedicated graphics card

Dataset:
3500 images
Annotated in Intel CVAT
Downloaded TFRecord from CVAT
